sale_coupon_multi_currency/tests/test_sale_coupon_multi_currency.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from .common import TestSaleCouponMultiCurrencyCommon
+
+
+class TestSaleCouponMultiCurrency(TestSaleCouponMultiCurrencyCommon):
+    """Test custom currency on program."""
+
+    def test_01_program_multi_currency(self):
+        """Set custom currency.
+
+        Case 1: set custom currency other than company currency.
+        Case 2: unset custom currency to use company currency.
+        """
+        # Case 1.
+        self.program_coupon_percentage.currency_id = self.currency_other.id
+        self.assertEqual(
+            self.program_coupon_percentage.currency_id, self.currency_other
+        )
+        self.assertEqual(
+            self.program_coupon_percentage.currency_custom_id, self.currency_other
+        )
+        self.assertEqual(self.company_main.currency_id, self.currency_company)
+        # Case 2.
+        self.program_coupon_percentage.currency_custom_id = False
```

Comment:
To have a behavior like in interface:
```suggestion
        self.program_coupon_percentage.currency_id = False
        self.program_coupon_percentage._inverse_currency_id()
```

Reply:
I already answered you this why it wont work on test. And there is no `currency_id =  False` on interface, because field is required.

Reply:
Ok, then forgot this comment.